docs/dev_process.md

```diff
@@ -0,0 +1,99 @@
+[Back to README](../README.md)
+
+## Background
+
+This document outlines the process to be followed - for Agile Ventures members - in developing and
+submitting a pull request for this project.  The same process is used
+regardless of whether the pull request is for a feature, a bug or a chore.
+
+We use Waffle as our story board for this project.  The waffle board for this
+project is [here](https://waffle.io/WikiEducationFoundation/WikiEduDashboard).
+
+Make sure you are added as a "collaborator" to the project - confirm with the AV coordinator if there is any question.
```

Comment:
Maybe we should add here how to be added as a collaborator.... i.e contact the slack channel